cmd/mattermost/commands/export.go

```diff
@@ -168,6 +171,11 @@ func buildExportCmdF(format string) func(command *cobra.Command, args []string)
 			return errors.New("exportFrom must be a positive integer")
 		}
 
+		endTime, err := command.Flags().GetInt64("exportTo")
+		if err != nil {
+			return errors.New("exportFrom flag error")
+		}
+
 		limit, err := command.Flags().GetInt("limit")
```

Comment:
Can we return an error if both `limit` and `endTime` are greater than `0`? Or at least we can log a warning that results will be inconsistent.

Reply:
I'd rather just warn about this. There may be a use case where you want to limit to a timeframe or number of posts, whichever is lower.

Reply:
Yeah but what if there are multiple posts with the same timestamp? Then you'll need to use the postid as a cursor right?

Reply:
Right. I'm not claiming to have a good use case for it but someone might. 2/5